server/datastore/mysql/migrations/tables/20231207102321_AddIndexSoftwareNameSourceBrowser.go

```diff
@@ -0,0 +1,27 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231207102321, Down_20231207102321)
+}
+
+func Up_20231207102321(tx *sql.Tx) error {
+	_, err := tx.Exec(`ALTER TABLE software_titles ADD UNIQUE INDEX idx_sw_titles (name, source, browser)`)
```

Comment:
curious why is there a different migration for this? can we group the index drop/creation in the same migration?

Reply:
I tried initially but ran into an error trying to do both in the same migration transaction.